pynicotine/gtkgui/ui/dialogs/fastconfigure.ui

```diff
@@ -257,7 +257,7 @@
             <child>
               <object class="GtkLabel">
                 <property name="visible">True</property>
-                <property name="label" translatable="yes">Nicotine+ uses peer-to-peer networking to connect to other users. In order to allow users to connect to you without trouble, an open listening port is crucial.</property>
+                <property name="label" translatable="yes">Nicotine+ uses peer-to-peer networking. An open port on your router — properly forwarded to your computer — is required.</property>
```

Comment:
Perhaps we should point out something that would motivate people to forward their ports, e.g. not being able to find as many files.

Reply:
Nicotine+ uses peer-to-peer networking to search for shared files. An open port that is properly forwarded from your router helps to distribute more results between users.

Reply:
I don't mind not being absolutely right in these lines. It is a terse description, aiming to tell you what you need to know rather than a bill which has to correctly describe all cases.

The same goes for the other sentence.

Reply:
`Nicotine+ uses peer-to-peer networking to connect to other Soulseek users.`

Further description about ports is not required, because this is outlined in a following paragraph on the page.

Reply:
I disagree, this is a great place to explain it.

Reply:
Can we move this change to another PR?